src/condor_daemon_core.V6/daemon_core.cpp

```diff
@@ -8916,12 +8916,60 @@ int extractInheritedSocks (
 	return cSocks;
 }
 
+#ifdef WIN32
+static char * redact_condor_inherit(char* redacted_env)
```

Comment:
What is the point of this input argument?

Reply:
it makes the scope of the variable outside the redact_ function.